src/main/java/com/acm/casemanagement/controller/UserController.java

```diff
@@ -87,7 +87,7 @@ public ResponseEntity<Void> deleteUser(@PathVariable Long id) {
         Optional<User> user = userService.getUserById(id);
```

Comment:
You have already added this logic in deleteById in userService and no need to do it 2 times.